doc/api/errors.md

```diff
@@ -1590,6 +1590,11 @@ A callback is almost always meant to only be called once as the query
 can either be fulfilled or rejected but not both at the same time. The latter
 would be possible by calling a callback more than once.
 
+<a id="ERR_MUSTCALL_UNAVAILABLE_DURING_EXIT"></a>
+### `ERR_MUSTCALL_UNAVAILABLE_DURING_EXIT`
+
+`mustCall()` was called within a `process.on('exit')` handler.
```

Comment:
```suggestion
`assert.mustCall()` was called within a `process.on('exit')` handler.
```

I would also add links to the documentation corresponding to both `assert.mustCall()` and `process.on('exit')`.